packages/www/hooks/use-api.tsx

```diff
@@ -76,9 +76,15 @@ const makeContext = (state: ApiState, setState) => {
       if (state.token && !headers.has("authorization")) {
         headers.set("authorization", `JWT ${state.token}`);
       }
-      const res = await fetch(`/api${url}`, {
+      const endpoint =
+        window.location.hostname !== "localhost" &&
```

Comment:
Can we maybe do this the other way and check `livepeerorg.vercel.app` and `livepeerorg.now.sh`, using livepeer.monster if they're present? Otherwise we'll break deployment to all other domains

Reply:
Yeah good call. All set!